BREAKING_CHANGE_AND_DEPRECATION.md

```diff
@@ -0,0 +1,46 @@
+# Breaking changes and deprecations
+
+Breaking changes are defined as a change to any of the following that causes installation errors or 
+unexpected runtime behavior after upgrading to the next stable minor version of Ratify(cli, plugin,):
+- Verification result and schema
+- Default configuration value
+- User facing plugin interfaces
+
+We do our best to maintain backwards compatibility, however breaking changes may be introduced between version for features that have not yet reached v1.0.0 or only available in RATIFY_EXPERIMENTAL flag.
```

Comment:
I feel RATIFY_EXPERIMENTAL is not the correct way for introducing breaking changes. RATIFY_EXPERIMENTAL features should be used for experimenting new compatible features? Since we did have some breaking changes in the past releases, maybe we could list existing breaking changes but try to strictly follow semantic versions in future release?

Reply:
as discussed, this is to state that we could introduce breaking changes to experimental features without changes